docs/user-guide/faq.md

```diff
@@ -115,3 +115,27 @@ Refer to [this](about-rules.md#-empty-line-before-and--max-empty-lines) section
 ## How do I "pin" to a particular branch, pull request or fork of stylelint in my `package.json`?
 
 Use [`npm-git-install`](https://github.com/lzrski/npm-git-install) to work around [a current limitation](https://github.com/npm/npm/issues/3055) of `npm`.
+
+## If I extend a configuration, will the options for each rule be merged or overridden?
+
+In most cases rules will be merged but **secondary options will be overriden**.
```

Comment:
Ok, I understand. I suggest we be explicit about this then.

Having said that, this is already explicitly stated within the [configuration](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#extends) guide. Perhaps we should just link, from within the FAQ, to this section of the docs like we do with the other FAQ entries?

Note, all options are overridden... not just the secondary ones.